napari/_qt/_tests/test_async_slicing.py

```diff
@@ -76,20 +108,20 @@ def test_async_slice_multiscale_image_on_pan(make_napari_viewer, qtbot, rng):
     image.corner_pixels = np.array([[0, 0, 0], [0, 4, 3]])
     image.events.reload(Event('reload', layer=image))
 
-    wait_until_vispy_image_data_equal(qtbot, vispy_image, data[1][1, 0:4, 0:3])
+    wait_until_vispy_image_data_equal(qtbot, vispy_layer, data[1][1, 0:4, 0:3])
```

Comment:
This is obnoxious, but I have no idea the proper way to go from the original to the expected. Can someone help?